configuration/analyzer_config.json

```diff
@@ -1003,9 +1019,9 @@
     "additional_config_params": {
       "api_key_name": "TRIAGE_KEY",
       "endpoint": "public",
```

Comment:
should add `analysis_type: "search"` parameter. Explicit is better than implicit.